hadoop-ozone/recon/src/main/resources/webapps/recon/ozone-recon-web/src/views/insights/om/om.tsx

```diff
@@ -818,26 +837,30 @@ export class Om extends React.Component<Record<string, object>, IOmdbInsightsSta
     // to fetch previous call stored all prevkey in array and fetching in respective tabs
     if (this.state.activeTab === '2') {
         this.setState({
-          prevKeyOpen: openPrevKeyList[openPrevKeyList.indexOf(this.state.prevKeyOpen)-2]
+          prevKeyOpen: prevKeyListMap.get(this.state.pageDisplayCount - 1),
+          pageDisplayCount : this.state.pageDisplayCount - 1
         }, () => {
           this.fetchOpenKeys(this.state.includeFso, this.state.includeNonFso, this.state.DEFAULT_LIMIT,this.state.prevKeyOpen);
         })
     } else if (this.state.activeTab === '3') {
       this.setState({
-        prevKeyDeletePending: keysPendingPrevList[keysPendingPrevList.indexOf(this.state.prevKeyDeletePending)-2]
+        prevKeyDeletePending: prevKeyListMap.get(this.state.pageDisplayCount - 1),
+        pageDisplayCount: this.state.pageDisplayCount - 1
       }, () => {
         this.fetchDeletePendingKeys(this.state.DEFAULT_LIMIT, this.state.prevKeyDeletePending);
       })
     } else if (this.state.activeTab === '4') {
       this.setState({
-        prevKeyDeleted: deletedKeysPrevList[deletedKeysPrevList.indexOf(this.state.prevKeyDeleted)-2]
+        prevKeyDeleted: prevKeyListMap.get(this.state.pageDisplayCount - 1),
+        pageDisplayCount : this.state.pageDisplayCount- 1
```

Comment:
nit
```suggestion
        pageDisplayCount : this.state.pageDisplayCount - 1
```

Reply:
Done Changes suggested by you in latest commit.